hazelcast/src/test/java/com/hazelcast/map/BasicMapTest.java

```diff
@@ -1191,6 +1194,26 @@ public void testIfWeCarryRecordVersionInfoToReplicas() {
     }
 
     @Test
+    public void github_11489() throws Exception {
+        HazelcastInstance hz = createHazelcastInstance();
```

Comment:
Has to be `HazelcastInstance hz = instances[1];`, since we already got a test factory.